src/services/Claudy.jsx

```diff
@@ -161,10 +161,13 @@ export class Claudy extends Component {
           <button className='coz-btn-close' onClick={onClose} />
           <button className='coz-claudy-menu-header-back-button' onClick={this.goBack} />
         </header>
-        {!emailStatus.isSending && emailStatus.isSent &&
-          <div className='coz-claudy-menu-action-description-success'>
-            {t('claudy.actions.support.success')}
-          </div>
+        { // FIXME: We use a `p` tag as long as a `div` will break the support
```

Comment:
Ping @goldoraf, we all saw this by our own eyes. Do you have a beginning of an explanation ?